crates/router/src/routes/metrics/utils.rs

```diff
@@ -1,12 +0,0 @@
-use std::time;
-
-#[inline]
-pub async fn time_future<F, R>(future: F) -> (R, time::Duration)
-where
-    F: futures::Future<Output = R>,
-{
-    let start = time::Instant::now();
-    let result = future.await;
-    let time_spent = start.elapsed();
-    (result, time_spent)
-}
```

Comment:
Deleted this in favor of the same function defined in the `common_utils` crate:

https://github.com/juspay/hyperswitch/blob/2123f63bc54030adf1d8f42f08240c71b03fc09c/crates/common_utils/src/metrics/utils.rs#L7-L17